src/vec_int128_ppc.h

```diff
@@ -2022,6 +2310,146 @@ vec_divsq_10e31 (vi128_t vra)
   return (vi128_t) result;
 }
 
+/** \brief Vector Divide Unsigned Double Quadword by const 10e31.
+ *
+ *  Compute the quotient of 256 bit value vra||vrb / 10e31.
+ *
+ *  \note vec_divudq_10e31() and vec_modudq_10e31() can be used
+ *  to perform long division of a multi-quaqword binary value by
+ *  the constant 10e31. The final remainder can be passed to
+ *  <B>Decimal Convert From Signed Quadword</B> (See vec_bcdcfsq()).
+ *  Long division it repeated on the resulting multi-quadword quotient
+ *  to extract 31-digits for each step. This is repeated until the
```

Comment:
"repeated" is repeated.

Reply:
I think we want:
 Long division is repeated on the resulting multi-quadword quotient
 to extract 31-digits for each step. This continued until the
 multi-quadword quotient is less than 10e31 which
 provides the highest order 31-digits of the of the multiple
 precision binary to BCD conversion.